src/snowflake/connector/connection.py

```diff
@@ -1182,6 +1183,11 @@ def __config(self, **kwargs):
             ]:
                 self._authenticator = auth_tmp
 
+        # read OAuth token from
+        if "token_file_path" in kwargs:
```

Comment:
nit: I think you can do `if kwargs.get("token_file_path"):` to achieve:

1.  check `token_file_path` is in kwargs
2. check if `token_file_path` in the kwargs is None

Reply:
done